ci/plans/ceramic.cue

```diff
@@ -68,6 +68,7 @@ dagger.#Plan & {
 
 	actions: {
 		_repo:        "js-ceramic"
+		_repo_composeDB: "composedb"
```

Comment:
Want to also align this with the rest of the naming convention?

e.g. `repo_ceramic` and `repo_cdb`

Reply:
oof, good call. Updated and rebased.